http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/message/HTTPMessageUtil.java

```diff
@@ -131,14 +131,18 @@ private static CarbonMessage setWebSocketCommonProperties(CarbonMessage cMsg,
      * Extract sender configuration from transport configuration.
      *
      * @param transportsConfiguration {@link TransportsConfiguration} which sender configurations should be extracted.
+     * @param isHttps
      * @return extracted {@link SenderConfiguration}.
      */
-    public static SenderConfiguration getSenderConfiguration(TransportsConfiguration transportsConfiguration) {
+    public static SenderConfiguration getSenderConfiguration(TransportsConfiguration transportsConfiguration,
```

Comment:
shall we get the scheme here and check it here rather than checking it in ballerina side?